### DISCUSSION TITLE
SPLITTING APPEND VALUE

### DISCUSSION THREAD

**Discussion by joelwebb** [Chia-Network/chia-blockchain#6288] [Support]
Recently after upgrading to 1.16, I have started seeing this crop up quite a bit while plotting --- It happens both in the GUI and in the Windows power shell: 

![splitting_append_value2](https://user-images.githubusercontent.com/25040566/120249641-8aae1e00-c249-11eb-8297-caf5da6963c3.PNG)
![splitting_append_value](https://user-images.githubusercontent.com/25040566/120249642-8b46b480-c249-11eb-82d7-e8e7515680d9.PNG)

Typically in power shell when this happens it stops the plotting but in the Gui some times the plots continue.

**OS:** Windows 10 Home - Version 20H2 - OS Build: 19042.985

**Hardware:** Threadripper 3990 - 64Cores 
126Gb Ram 2666hz Corsair
ASUS Zenith Extreme Motherboard
Boot SSD - 250GB Sabrent Rocket
1800W EVGA PSU
A lot of other HDDs and SSDs for plotting
No overclocking on the machine 

Some plots will successfully finish, while others will error out with what other people on the forum keep telling me is a ram issue: 
![unknown](https://user-images.githubusercontent.com/25040566/120249776-f8f2e080-c249-11eb-870e-e414ba62f6bb.PNG)

If I roll back to 1.14 or 1.15 now I still see these errors but I didn't see them prior to the 21th after installing the latest version.

Any ideas on what I can do to fix this?

**Comment by BrandtH22** [comment]
@joelwebb , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
